config/initializers/postgres_required_versions.rb

```diff
@@ -0,0 +1,8 @@
+ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.prepend Module.new {
+  def initialize(*args)
+    super
+    if postgresql_version < 90400 || postgresql_version >= 90600
+      raise "The version of PostgreSQL being connected to is incompatible with ManageIQ."
```

Comment:
Do we need to productize this?

Reply:
I was wondering the same thing 😬  Is there a mechanism (i18n, whatever) that we use for such things?

Reply:
`I18n.t("product.name")` I think should do it.

Reply:
If it was just in the logs I think we wouldn't need to but because it's a raise it could get to the UI I suppose?

Reply:
I agree. I think the idea was to be an in-your-face error and not just a warning. I'll add that for sure.